docs/detections/rules-ui-create.asciidoc

```diff
@@ -517,7 +517,11 @@ minutes at 14:03 but it does not run until 14:04, it will run again at 14:09.
 [[rule-action-variables]]
 ==== Alert notification placeholders
 
-You can use http://mustache.github.io/[mustache syntax] to add the following placeholders to <<rule-notifications, rule action>> fields:
+You can use http://mustache.github.io/[mustache syntax] to add the following placeholders to <<rule-notifications, rule action>> fields. Different variables are provided based on the rule's action frequency. 
+
+If the rule’s action frequency is a summary of alerts, it passes the variables that summarize the new, ongoing, and recovered alerts. Refer to {kibana-ref}/rule-action-variables.html#alert-summary-action-variables[Action frequency: Summary of alerts] more information about summary of alert variables. 
+
+If the rule’s action frequency is not a summary of alerts, it passes the following variables:
```

Comment:
In security solution we use all the context variables `{{context.*}}` and `{{state.signals_count}}` for `summary of alerts` as well.